DEVELOPER_GUIDE.md

```diff
@@ -141,6 +141,27 @@ Note - it may take a couple minutes to generate all the necessary bundles. If th
 [success][@osd/optimizer] 28 bundles compiled successfully after 145.9 sec, watching for changes
 ```
 
+### Run OpenSearch using Docker (Recommended for Mac users)
+
+Mac users are advised to set up OpenSearch and OpenSearch Dashboards using [Docker]((https://github.com/opensearch-project/OpenSearch-Dashboards/blob/main/docs/docker-dev/docker-dev-setup-manual.md)). Follow the instructions in the Docker Developer Guide to set up OpenSearch using Docker.
+
+### Run OpenSearch Dashboards for Mac Users
+
+1. After setting up OpenSearch using Docker, make sure that the OpenSearch server instance is up and running. You can validate this by running curl localhost:9200 in another console tab or window ([see OpenSearch developer guide]((https://github.com/opensearch-project/OpenSearch/blob/main/DEVELOPER_GUIDE.md#run-opensearch))).
```

Comment:
I assume you mean the instructions in https://github.com/opensearch-project/OpenSearch-Dashboards/blob/main/docs/docker-dev/docker-dev-setup-manual.md. If that s the case can you add this section there? Also can you reword it as a fallback option if the steps mentioned in the original readme dont work.